charts/internal/shoot-system-components/charts/csi-driver-node/templates/_helpers.tpl

```diff
@@ -11,9 +11,5 @@ cinder.csi.openstack.org
 {{- end -}}
 
 {{- define "csi-driver-node.storageversion" -}}
-{{- if semverCompare "<= 1.18.x" .Values.kubernetesVersion -}}
```

Comment:
Can https://github.com/gardener/gardener-extension-provider-openstack/blob/024f99d4dc119437a999beac6c6763e4ce539c45/pkg/controller/controlplane/valuesprovider.go#L755 be now removed?